js/scripts/utils.ts

```diff
@@ -2,9 +2,13 @@
 // Licensed under the MIT License.
 
 import { WriteStream } from 'fs';
+import {bootstrap as global_agent_bootstrap} from 'global-agent';
 import * as https from 'https';
 import { JSZipObject } from 'jszip';
 
+// Bootstrap global-agent to honor the proxy settings
```

Comment:
can add more few comments referring to GLOBAL_AGENT_HTTPS_PROXY or documents

Reply:
Done.